src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-msbuild.cmd

```diff
@@ -0,0 +1,40 @@
+@echo off
+setlocal
+
+set DOTNET_CMD=%~1
+set MSBUILDRUNTIME_DIR=%~2
+set TOOLSDIR=%~3
+set BUILDTOOLS_PACKAGE_DIR=%~dp0
+set MSBUILD_CONTENT_JSON={"dependencies": {"Microsoft.Portable.Targets": "0.1.1-dev"},"frameworks": {"dnxcore50": {},"net46": {}}}
+
+if not exist "%DOTNET_CMD%" (
+  echo ERROR: Cannot find dotnet cli at [%DOTNET_CMD%]. Please pass in the path to dotnet.exe as the 1st parameter.
+  exit /b 1
+)
+
+if [%MSBUILDRUNTIME_DIR%]==[] (
+  echo ERROR: MSBuild Runtime Directory is required for the restore. Please pass in the path where MSBuild will be restored as the 2nd parameter.
+  exit /b 1
+)
+
+if not exist "%TOOLSDIR%" (
+  echo ERROR: Cannot find the buildtools location at [%TOOLSDIR%]. Please pass in the path to buildtools as the 3rd parameter.
+  exit /b 1
+)
+:: Restore most of the runtime
+cd "%BUILDTOOLS_PACKAGE_DIR%\msbuild-runtime\"
+call "%DOTNET_CMD%" restore --source https://www.myget.org/F/roslyn-nightly/ --source https://www.myget.org/F/dotnet-core/ --source https://www.myget.org/F/dotnet-buildtools/ --source https://www.nuget.org/api/v2/
```

Comment:
right, as I said in a previous comments, this is not up to date. Basically what I did is to add the msbuild runtime into the tool-runtime/project.json so that the Tools directory has everything it needs includding .Net Core MSBuild. Once I have a ready version of buildtools, I'll push all of those updates into this branch.